app/views/conversations/show.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="chatboxcontent">
+    <% if @messages.any? %>
+        <%= render @messages %>
```

Comment:
You don't need the if..any here. If there aren't any, render messages won't render anything.